org.eclipse.winery.workflowmodeler/src/app/app.component.html

```diff
@@ -0,0 +1,17 @@
+<!--
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+-->
+
+<router-outlet></router-outlet>

```

Comment:
Instead of the `<router-outlet></router-outlet>` we could directly add the `<b4t-toolbar></b4t-toolbar>` here...